server/pulp/server/auth/authorization.py

```diff
@@ -1,19 +1,5 @@
 # -*- coding: utf-8 -*-
 
-# Copyright © 2010-2011 Red Hat, Inc.
-#
-# This software is licensed to you under the GNU General Public
-# License as published by the Free Software Foundation; either version
-# 2 of the License (GPLv2) or (at your option) any later version.
-# There is NO WARRANTY for this software, express or implied,
-# including the implied warranties of MERCHANTABILITY,
-# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
-# have received a copy of GPLv2 along with this software; if not, see
-# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
-
-"""
-Utility functions to manage permissions and roles in pulp.
-"""
```

Comment:
PEP-257 requires modules to have docblocks too, and once I finish PEP-8, PEP-257 is next. Plz to add back ☺